cmd/certificates/setup.go

```diff
@@ -40,34 +38,24 @@ func cmdSetup(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	if !config.Overwrite && !valid {
+	if !setupCfg.Overwrite && !valid {
 		fmt.Printf("certificate signer configuration already exists (%v). rerun with --overwrite\n", setupDir)
 		return nil
 	}
 
-	if config.Overwrite {
-		config.CA.Overwrite = true
-		config.Identity.Overwrite = true
-		config.Signer.Overwrite = true
+	if setupCfg.Overwrite {
+		setupCfg.Signer.Overwrite = true
 	}
 
-	if _, err := config.Signer.NewAuthDB(); err != nil {
+	if _, err := setupCfg.Signer.NewAuthDB(); err != nil {
 		return err
 	}
 
-	status, err := config.Identity.Status()
```

Comment:
is this not needed now? shouldn't this be just refactored to use `setupCfg.Identity`?

Reply:
Prevoiusly, `config.Identity` was an `identity.SetupConfig`, which implements `#Status`, but the only place we're using `SetupConfig` since we separated the identity tool from everything is there. `certificates run` will now fail the same way every other cli command fails when it's missing an identity.